bot/exts/moderation/slowmode.py

```diff
@@ -58,6 +58,10 @@ async def set_slowmode(self, ctx: Context, channel: Optional[TextChannel], delay
             log.info(f'{ctx.author} set the slowmode delay for #{channel} to {humanized_delay}.')
 
             await channel.edit(slowmode_delay=slowmode_delay)
+            if channel.id == Channels.python_general:
```

Comment:
Instead of restricting it to just python general and manually adding it in future for other channels, why not create a list like `!otn`, and the moderators can add channels to the list and remove it.

Reply:
This would make this unnecessary complex. But I still think discord.py channel should be added at least.